examples/motoko_examples/http_counter/package.json

```diff
@@ -0,0 +1,17 @@
+{
+  "name": "http_counter",
+  "version": "0.1.0",
+  "description": "Reimplementation of the [Motoko Http Counter example](https://github.com/dfinity/examples/tree/master/motoko/http_counter)",
+  "main": "index.js",
```

Comment:
If you could copy the other examples' package.json files, let's remove any fields that the others do not have (like main) and set the version similar to other versions